### DISCUSSION TITLE
Scratch 3.0 Block Layout horizontal

### DISCUSSION THREAD

**Discussion by mincubo** [ScratchAddons/ScratchAddons#7281] [Ideas]
add the old Scratch 3.0 Block layout like 

![Scratch_3 0_Block_Layout_Comparison](https://github.com/ScratchAddons/ScratchAddons/assets/152664689/b4bd60ec-613f-4365-b7fd-ef22dc9b0767)

**Comment by BroJac5246** [comment]
Umm ever heard of [Scratch Jr.](https://play.google.com/store/apps/details?id=org.scratchjr.android)?

This would be super impractical — imagine how custom blocks would have to look.

**Reply by mincubo** [reply]
no one like scratch jr

**Reply by BroJac5246** [reply]
Yeah that's true.

But seriously, this doesn't make sense — it's super impractical.

(Also it would be very difficult to program this addon)

**Comment by Samq64** [comment]
Duplicate of #4039
